recipes/default.rb

```diff
@@ -6,31 +6,49 @@
 #
 # All rights reserved - Do Not Redistribute
 #
-
 include_recipe 'ark::default'
 
-# Download and build the selected version of SnapRaid
-ark 'snapraid' do
```

Comment:
It's still there! Just further down the page. I'm going to split it out into separate recipes like you suggest though.